js/validation.js

```diff
@@ -0,0 +1,68 @@
+const valMsg = document.getElementById('validation_msg');
+const inputs = document.querySelectorAll('form input, form textarea');
+let clearMsgTimeOut = null;
+
+// Validation function //
+
+function validationHandler(e) {
+  e.preventDefault();
+
+  let emailOk = false;
+  const textOkArr = [];
+  const msgArr = [];
+
+  inputs.forEach((item) => {
+    const { name } = item;
+
+    switch (name) {
+      case 'email':
+        emailOk = /^([a-z0-9-_.]+)@(\w+)\.(\w+)$/i.test(item.value);
+        if (emailOk) {
+          item.classList.remove('invalid');
+        } else {
+          item.classList.add('invalid');
+          msgArr.push('Invalid email format');
+        }
+        break;
+      case 'name':
+        if (item.value !== '') {
+          textOkArr.push(true);
+          item.classList.remove('invalid');
+        } else {
+          item.classList.add('invalid');
+          textOkArr.push(false);
+          msgArr.push('This field should not be empty');
+        }
+        break;
+      case 'description':
+        if (item.value !== '') {
+          textOkArr.push(true);
+          item.classList.remove('invalid');
+        } else {
+          item.classList.add('invalid');
+          textOkArr.push(false);
+          msgArr.push('This field should not be empty');
+        }
+        break;
+      default:
+        break;
+    }
+  });
+
+  clearTimeout(clearMsgTimeOut);
+  valMsg.innerText = msgArr.join('\n');
+  clearMsgTimeOut = setTimeout(() => {
+    valMsg.innerText = '';
+  }, 2000);
+
+  if (emailOk && textOkArr.includes(false) === false) {
+    fetch('https://formspree.io/f/mjvdnklq', {
+      method: 'POST',
+      body: new FormData(document.querySelector('form'))
+    });
+
+    alert('Your information has been succesfully delivered!');
+  }
+}
```

Comment:
- [ ] `required`
- [ ] the project requirements states that 
`When the user submits the form, you check if the email is in lower case.
If the validation is OK, the form is sent.
If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.`
However, your project is allowing to submit a None lower case email 😞 
please make amends 🥂